scanpy/experimental/pp/_highly_variable_genes.py

```diff
@@ -199,7 +199,7 @@ def _highly_variable_pearson_residuals(
 
         sums_genes = np.array(X_batch.sum(axis=0)).ravel()
         sums_cells = np.array(X_batch.sum(axis=1)).ravel()
-        sum_total = np.sum(sums_genes).ravel()
+        sum_total = np.sum(sums_genes)
```

Comment:
this makes a 0D array into a 1D array. Later a 0D array is expected, so let’s just not do that.